src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Parent class of Address, Email and Phone
```

Comment:
the comment shouldn't talk about the child classes. 
header comments can include what it is, how it is used but shouldn't include where it should be used